examples/new/types.ts

```
@@ -0,0 +1,148 @@
+        : never;
+
+export type TemplateVariables<T extends string> = Pretty<{
+    [K in ExtractTemplateVariables<T>]: string;
+}>;
+
+export type AgentMemory = {
+    working: WorkingMemory;
+    shortTerm: {};
+};
+
+export type Expert = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)